pkg/exporter/sumologicexporter/factory_test.go

```diff
@@ -48,6 +48,7 @@ func TestCreateDefaultConfig(t *testing.T) {
 		SourceHost:         "",
 		Client:             "otelcol",
 		GraphiteTemplate:   "%{_metric_}",
+		TransformMetadata:  true,
```

Comment:
why not `DefaultTransformMetadata` instead of `true`? ;)

Reply:
The other settings are also set not using the `Default` fields, like `CompressEncoding`, `LogFormat`, `MetricFormat` etc.

I believe the purpose of this test is to verify that the default config sets the values we expect. If the default value is changed, this test should be updated.